lab-michael/karma.conf.js

```diff
@@ -0,0 +1,28 @@
+const webpackConfig = require('./webpack.config.js');
+webpackConfig.entry = {};
```

Comment:
You need to delete the `webpackConfig.entry` object, rather than set it to an empty obj. This is a change with Webpack 2.x